.flake8

```
@@ -0,0 +1,4 @@
+[flake8]
+extend-ignore = E203, E266, E501, C901, F401
```

```suggestion
extend-ignore = E203,E266,E501,C901,F401
```

Don't really know if that would make a difference

---

Yeah I think so, I had to make the max_complexity higher and C901 was supposed to be ignored.